chart/skywalking/Chart.yaml

```diff
@@ -39,6 +39,7 @@ dependencies:
     repository: https://raw.githubusercontent.com/bitnami/charts/archive-full-index/bitnami
     condition: postgresql.enabled
   - name: skywalking-banyandb-helm
-    version: 0.1.0
-    repository: oci://registry-1.docker.io/apache
+    alias: banyandb
+    version: 0.0.0-8bdff75
+    repository: oci://ghcr.io/apache/skywalking-banyandb-helm
```

Comment:
You should point to docker hub, rather than ghcr. Why is this changed?

Reply:
Could you release 0.2.0 of the banyandb helm?

Reply:
Meanwhile, where is ` oci://registry-1.docker.io/apache`? Is this repository existing?

Reply:
We have not yet released banyandb and banyandb-helm, so I temporarily pointed the repository to ghcr. I will update these values together after the releases are completed.

Reply:
We have the previous(0.1.0) version, https://github.com/apache/skywalking-banyandb-helm.

Are you going to release that soon? Otherwise, once skywalking-helm 4.6 is releasing, it will carry the unreleased BanyanDB version(some snapshot version). This is not correct in the ASF.

Reply:
As far as I know, we will release shortly after the completion of testing on banyandb, and it should be relatively fast.

Reply:
Could you post an issue on Helm 4.6.0 milestone to track this, and you should fix [this](https://github.com/apache/skywalking-helm/pull/145#issuecomment-2021862061)

Reply:
@hanahmily told me, your deployment should not care about BanyanDB 0.5 or 0.6(dev or unreleased). So, this should not be a blocker for you. 
I know 0.6 is much better, so you only need to keep all in 0.5 by default, and upgrade to 0.6 once it is released.

Reply:
Okay.

Reply:
> @hanahmily told me, your deployment should not care about BanyanDB 0.5 or 0.6(dev or unreleased). So, this should not be a blocker for you. I know 0.6 is much better, so you only need to keep all in 0.5 by default, and upgrade to 0.6 once it is released.

There is a [feature](https://github.com/apache/skywalking-banyandb/pull/331) related to etcd authentication that has not been implemented in version 0.5. It seems that we still need to rely on the upcoming BanyanDB 0.6 version. After discussing with @hanahmily , we decide to change these tags later on.